Weekly Security Link Dump (Week of May 18th)

cneill opened the pull request:
__Bio:__
Charles Neill is a Security Engineer at Rackspace, who tests Rackspace products as well as OpenStack projects like Barbican. He is also a developer on [DefectDojo](https://github.com/rackerlabs/django-DefectDojo), an open source defect tracking system using the Django framework.

__Social Media:__
Twitter: [@ccneill](https://twitter.com/ccneill)
Github: [cneill](https://github.com/cneill)
Website: [inventropy.us/blog/](https://inventropy.us/blog/)

__Editing Preference:__
Feel free to edit as needed.

__Posting Preference:__
Desired date to publish: May 19th, 2015

Thanks!

catlook commented on src/site_source/_posts/2015-05-18-weekly-security-link-dump.md:
```diff
@@ -0,0 +1,52 @@
+
+# Reference / Tutorials
+
+- [__A week with a Rails security strategy__][rails] - This article presents a nice framework for taking care of the security of your products on a weekly basis. It is geared towards Rails, but you can replace a lot of the Rails-specific pieces with the applicable tools for the language you're using. It includes simple processes like checking for outdated packages that might have security issues  ...
```
... but it also contains more holistic ...

catlook commented on src/site_source/_posts/2015-05-18-weekly-security-link-dump.md:
```diff
@@ -0,0 +1,52 @@
+
+# Random Link of the Week
+
+- [__Lily__][lily] - Ever wanted a drone to capture all your motions on HD video? Just throw this thing in the air and it will fly around tracking you and shooting video automatically. Personally, I'm not looking forward to the day we have thousands of these things flying around everywhere capturing everything we do, but at least they'll look cool.
```
... in the air, and it ...

catlook commented on src/site_source/_posts/2015-05-18-weekly-security-link-dump.md:
```diff
@@ -0,0 +1,52 @@
+[air_gap]: http://en.wikipedia.org/wiki/Air_gap_%28networking%29
+[yubikey]: https://www.yubico.com/products/yubikey-hardware/
+
+[lily]: https://www.lily.camera/
```
All done editing :)

cneill commented:
Thanks, Cat! Just made the suggested edits. Let me know if there's anything else that needs a change.

I don't know if y'all typically share this information, but I'd be curious to see what the traffic is like on these two posts to determine whether it's worth doing weekly, or if we should make it a monthly thing and/or focus on other topics.

rgbkrk commented on src/site_source/_posts/2015-05-18-weekly-security-link-dump.md:
```diff
@@ -0,0 +1,52 @@
+    - Security
+---
+
+Not a big update this week, but there are definitely some interesting links in here. If you have any feedback, please let me know. Hope you enjoy!
```
I'd prefer a real summary here. Otherwise this sounds like you just slapped a bunch of stuff together.

cneill commented:
Sorry about the lack of a real summary to start with, but I've updated it to give a summary of the links in the body, and updated some links to give more information. Let me know if there's anything else you'd like to see changed.

Thanks!

rgbkrk commented on src/site_source/_posts/2015-05-18-weekly-security-link-dump.md:
```diff
@@ -0,0 +1,62 @@
+After several [high-profile][deprecation] [articles][deprecation2] [calling][deprecation3] for the deprecation of plaintext HTTP recently, the new HTTP/2 RFC has arrived! While it doesn't explicitly mandate SSL/TLS, HTTP/2 definitely has some improvements from a security perspective. Unfortunately, the Open Smart Grid Protocol has not had such good news this week, as researchers uncovered serious ...
+
+If you have any feedback about this week's links, the format of these posts, or anything else, please [let me know][twitter]!
+
```
That's so much better. Thank you.

catlook replied:
Looks good to me too :)

rgbkrk commented:
:shipit:
